pkg/kubelet/cm/containermap/container_map.go

```diff
@@ -20,23 +20,35 @@ import (
 	"fmt"
 )
 
-// ContainerMap maps (containerID)->(*v1.Pod, *v1.Container)
-type ContainerMap map[string]struct {
+// cmItem (ContainerMap ITEM) is a pair podUID, containerName
+type cmItem struct {
 	podUID        string
 	containerName string
 }
 
+// ContainerMap maps (containerID)->(podUID, containerName)
+type ContainerMap map[string]cmItem
+
 // NewContainerMap creates a new ContainerMap struct
 func NewContainerMap() ContainerMap {
 	return make(ContainerMap)
 }
 
+// Clone creates a deep copy of the ContainerMap
+func (cm ContainerMap) Clone() ContainerMap {
```

Comment:
This can be replaced with `maps.Copy`.

Reply:
TIL, thanks! Do you mind if I post a followup cleaning later? I'd like to unblock CI first?

Reply:
Okay.

Reply:
As promised: https://github.com/kubernetes/kubernetes/pull/128679